docs/faq.rst

```diff
@@ -693,6 +693,85 @@ Additional reading:
   * `How to avoid NaN gradients when using where <https://github.com/tensorflow/probability/blob/master/discussion/where-nan.pdf>`_.
 
 
+Gradients are zero for rank-based functions
+-------------------------------------------
+
+If you define a function that processes the input using rank-based operators
```

Comment:
I don't know what "rank-based" means :(

Reply:
I tried to give examples later in the sentence – anything I could do to make that more clear?

Reply:
To elaborate, what I had in mind is functions like `greater`, `max`, `argsort`, etc. whose output depends on comparing the order/ranking of inputs.

Reply:
Ooh I was thinking rank like the number of dimensions. I think "sort-order-based functions" would be clearer to me, and/or put "whose output depends on comparing the sort-order/ranking of inputs" around here.

Reply:
Thanks - done. PTAL!